### PR TITLE
SEP-6 new flow

### PR DESCRIPTION
**Pull request by quietbits**

### CODE REVIEW

File: packages/demo-wallet-client/src/components/Sep6/Sep6Deposit.tsx [hunk 1]
```diff
@@ -360,6 +365,26 @@ export const Sep6Deposit = () => {
...
+        <Modal.Body>
+          <p>Submit the deposit.</p>
+        </Modal.Body>
+
+        <Modal.Footer>
+          <Button onClick={() => dispatch(sep6DepositAction())}>Submit</Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  }
```
Comment (quietbits) [inline packages/demo-wallet-client/src/components/Sep6/Sep6Deposit.tsx:386]: We need this extra step to trigger transaction polling after the SEP-12 fields are submitted. We can change the copy of the modal, and the same goes for withdrawal.

**Review by philipliu — COMMENTED** [review]
LGTM other than the deposit instructions modal.

**Review by philipliu — APPROVED** [review]

File: packages/demo-wallet-client/src/components/Sep6/Sep6Deposit.tsx (hunk 1 above)
Comment (philipliu) [reply packages/demo-wallet-client/src/components/Sep6/Sep6Deposit.tsx:386]: Will this be added in a different PR?

Comment (quietbits) [reply packages/demo-wallet-client/src/components/Sep6/Sep6Deposit.tsx:386]: This is already implemented. It looks like it's conflicting with the optional `instructions` for deposit. I'm looking into that.

Comment (philipliu) [reply packages/demo-wallet-client/src/components/Sep6/Sep6Deposit.tsx:386]: This is an issue with AP's implementation. The transaction moves into the completed status before the wallet polls and sees the `pending_user_transfer_start` status.

**Comment by JakeUrban** [comment]
Are we going to add the SEP-38 functionality in this PR or a subsequent one?

**Comment by JakeUrban** [comment]
@quietbits and I talked offline and decided to merge this PR separately from adding quotes, we'll do that next.

**Pull request merged**